compiler/compile/VirtualGuard.cpp

```diff
@@ -227,38 +227,31 @@ TR_VirtualGuard::createBreakpointGuardNode
 #ifdef J9_PROJECT_SPECIFIC
 /*
  * Shape of a breakpoint guard:
- * iflcmpeq/ificmpeq
- *    land/iand
- *       lloadi/iloadi <ConstantPool shadow>
- *          aconst J9Method
+ * ificmpne
+ *    iand
+ *       bu2i
+ *          buloadi <ConstantPool shadow>
```

Comment:
buloadi - ie byte unsigned load indirect - is deprecated; we should be using bloadi since loading a byte width has no sign and then the bu2i conversion will handle the sign extension when widening to integer. In theory this could be a band and ifbcmpne and bconst but I'm not sure how well they would be handled by the optimizer so having an i and living with the zero extension for symmetry with other tests could make sense.